server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -135,10 +204,30 @@ export class MongoCollection<T> implements core.ICollection<T> {
 
         await this.collection.updateMany(filter, update, options);
     }
+
+    private async requestWithRetry<TOut>(request: () => Promise<TOut>, callerName: string): Promise<TOut> {
+        return requestWithRetry<TOut>(
+            request,
+            callerName,
+            {}, // telemetryProperties
+            (e) => this.retryEnabled && this.mongoErrorRetryAnalyzer.shouldRetry(e), // ShouldRetry
+            3, // maxRetries
+            1000, // retryAfterMs
+            (error: any, numRetries: number, retryAfterInterval: number) =>
```

Comment:
Make 3 and 1000 constants like MaxFetchSize?

Reply:
Addressed in d4556832bfd0a9c0cb5bde9bbacb337ab63974ee